packages/abi-utils/lib/arbitrary.test.ts

```diff
@@ -0,0 +1,55 @@
+import { testProp } from "jest-fast-check";
+import * as abiSchema from "@truffle/contract-schema/spec/abi.spec.json";
+import { matchers } from "jest-json-schema";
+
+expect.extend(matchers);
+
+import * as Arbitrary from "./arbitrary";
+
+const withDefinitions = (schema: object) => ({
```

Comment:
This function probably warrants a comment at least (I figured out what it does, but it could still use explanation).